Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Standard Dockerfile for GOV.UK services. Random service-specific stuff should *not*
+# go in this file. Instead, you should copy this file into the directory for the
```

Comment:
Unsure about this advice. Can't we tell people to import the image?

Reply:
Agreed, this isn't ideal. I've documented the 'single bloated image/Dockerfile' on the card as something to think about more.